### DISCUSSION TITLE
How often do you update fakerjs.dev site?

### DISCUSSION THREAD

**Discussion by minhtaile2712** [faker-js/faker#1904] [Q&A] [answered]
Please update the fakerjs.dev site to reflect the latest fix/updates from the repo.
I see some errors have been fixed long time ago but the site is still the same as nothing has ever been touched.

**Comment by matthewmayer** [comment]
https://fakerjs.dev/ has documentation for the latest stable release (v7.x.x).
If you want documentation for the latest alpha version (v8.x.x) its at https://next.fakerjs.dev/

**Comment by minhtaile2712** [comment, marked as answer]
In this section [Usage/Create complex objects](https://fakerjs.dev/guide/usage.html#create-complex-objects), at the third code snippet, the code line
```typescript
const sex = this.faker.name.sexType();
```
should be fixed as
```typescript
const sex = faker.name.sexType();
```
which has been fixed in `next` branch. We should fix this in the stable documentation v7.x.x too.

**Reply by Shinigami92** [reply]
Would you like to contribute and fix that?

All you need is to checkout the https://github.com/faker-js/faker/tree/v7 branch and open a PR on that.
When opening the PR in GitHub, you need to select "target branch" v7.

When it is ready to merge, one of the admins (e.g. me), need to unlock the v7 branch, merge the PR and lock it again.

**Reply by minhtaile2712** [reply]
Thank you for your instruction. I have created a new pull request. I'm glad to contribute to this project.

**Reply by ST-DDT** [reply]
FFR: 

- https://github.com/faker-js/faker/pull/1906
